Jenkinsfile

```diff
@@ -1,5 +1,11 @@
-buildPlugin(useContainerAgent: true, configurations: [
-    [ platform: "linux", jdk: "11"],
-    [ platform: "linux", jdk: "17"],
-    [ platform: "windows", jdk: "11"]
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/
+buildPlugin(
+  forkCount: '1C', // run this number of tests in parallel for faster feedback.  If the number terminates with a 'C', the value will be multiplied by the number of available CPU cores
+  useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
+  configurations: [
+    [platform: 'linux', jdk: 21],
+    [platform: 'windows', jdk: 17],
```

Comment:
I am still baffled why this is the recommended setting. From a cost perspective sure. But I believe there should at least be one combination where just one variable is different.

Reply:
This is not the place to discuss it; just copying the `Jenkinsfile` from `archetypes`.